examples/Demo/FluentUI.Demo.Client/Infrastructure/AppVersionService.cs

```diff
@@ -0,0 +1,35 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using System.Reflection;
+
+namespace FluentUI.Demo.Client;
+internal class AppVersionService : IAppVersionService
+{
+    public string Version
+    {
+        get => GetVersionFromAssembly();
+    }
+
+    static public string GetVersionFromAssembly()
```

Comment:
Can you add a comment to explain which version will be used and which string format is returned?

Reply:
Changed this so one would use AppVersion.Version. Added comment there (this is now private)